src/main/java/uk/gov/hmcts/darts/common/repository/UserAccountRepository.java

```diff
@@ -35,8 +35,9 @@ public interface UserAccountRepository extends
         WHERE courthouse = :courthouse
         AND securityRole.id = :securityRole
         AND userAccount.active = true
+        and userAccount not in :excludingUsers
```

Comment:
Maybe check against null although maybe we this will work regardless

Reply:
I don't think this can ever be null just an empty set which will get ignored